dcat/index.html

```diff
@@ -3504,7 +3474,7 @@ <h2>How to specify dataset series</h2>
 <h2>Property values inheritance in dataset series</h2>
 
 <aside class="ednote">
-<p>The approach proposed in this section is being reviewed.</p>
+<p>The approach proposed in this section is being reviewed.  </p>
```

Comment:
@andrea-perego , @agbeltran : I wonder if we want to keep this editorial note?

Reply:
I would drop it, and include instead the reference to https://github.com/w3c/dxwg/issues/1273